dvc/exceptions.py

```diff
@@ -178,8 +178,9 @@ def __init__(self, stages):
 class ConfirmRemoveError(DvcException):
     def __init__(self, path):
         super(ConfirmRemoveError, self).__init__(
-            "unable to remove '{}' without a confirmation from the user. Use "
-            "'-f' to force.".format(path)
+            "unable to remove '{}' without a confirmation from the user."
+            " Use '--force' (repro) or '--force-remove-outs' (run) to confirm"
```

Comment:
Do we need to check that on `repro` though? Didn't we agree that it is allowed for `repro` to do that silently? `--force` for `repro` has a totally different meaning than `--force-remove-outs`.

Reply:
:thinking: that's true, @efiop , when `--force` is used with `repro` it reproduces the stage even if the stage didn't _change_.

I'm not sure why the error message was like that.

Reply:
It might be a bug:
```python
    def run(self, dry=False, no_commit=False, force=False):
        if (self.cmd or self.is_import) and not self.locked and not dry:
            self.remove_outs(ignore_remove=False, force=False)
```

I think it should have been `self.remove_outs(ignore_remove=False, force=force)`